src/sentry/static/sentry/app/views/settings/organizationFiltersAndSampling/index.tsx

```diff
@@ -25,7 +25,9 @@ const Index = ({organization}: Props) => (
       />
     )}
   >
-    <OrganizationFiltersAndSampling />
+    <OrganizationFiltersAndSampling
+      organization={{...organization, dynamicSampling: []}}
```

Comment:
Is this just temporary? Will `dynamicSampling` be a property of the organization?

Reply:
yes it's everything under a feature flag and work in progress ... @RaduW  did not merge his PR yet https://github.com/getsentry/sentry/pull/23028